meteor-app/imports/ui/visit/index.js

```diff
@@ -13,6 +13,7 @@ const debug = require('debug')('b2b:visit')
 
 export default withTracker(props => {
   const id = props.match.params.id
+  console.log(id)
```

Comment:
I'm not a fan of console.log, particularly where you are just logging an id like this without any context. If you leave this code in place, the server console will be outputting id's merrily, and you will have no idea where it came from. 

If you really need to be outputting info, you can use `debug()` which does the same, but automatically prefixes the information with the context information, it's also a good idea to provide some local context as well, like this:

```
debug(`memberId: ${id}`)
```

It also won't output anything in production

Reply:
All good. I've removed the console.logs. I think it was used purely for testing. I'll start using debug() going forward and provide some context.